kolibri/core/assets/src/views/PaginatedListContainer.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+
+  <div>
+    <KGrid>
+      <KGridItem sizes="3, 3, 3">
+        <slot name="otherFilter"></slot>
+      </KGridItem>
+      <KGridItem sizes="9, 9, 9" align="right" class="text-filter">
+        <KFilterTextbox
+          v-model.trim="filterInput"
```

Comment:
in the past, using 'trim' has caused us issues. Is the use here intentional?

Reply:
this was the existing implementation. i'll just remove it